.github/ISSUE_TEMPLATE/---found-a-bug-and-i-solved-it.md

```diff
@@ -0,0 +1,24 @@
+---
+name: "\U0001F41E Found a Bug!"
+about: I found a problem! It is Jina’s bug and I’d like to bring it to others' attention.
+title: ''
+labels: kind/bug
+assignees: ''
+
+---
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is. -->
+
+**Describe how you solve it**
+<!-- copy past your code/pull request link -->
+
+---
+
+<!-- Optional, but really help us locate the problem faster -->
+
+**Environment**
+<!-- Run `jina --version-full` and copy paste the output here -->
+
+**Screenshots**
```

Comment:
Screenshots and logs. Also ask them to attach log files if they have any (`&>` is particularly useful here - pipes ALL output, not just `stdout` OR `stderr` to file - e.g. `python app.py index &> log.txt`)

Reply:
(This applies to all issue types that may need logs)

Reply:
I feel that `&>` is not relevant for this project.

Error, warning logs generally occurs in browser console. It would be harder to write that to a file. What I've seen most people doing is to copy from developer console.

Reply:
Ah, okay. Normally I've been running everything from terminal